tests/model_connectors/test_teaser.py

```diff
@@ -92,16 +93,22 @@ def test_teaser_single(self):
             Coupling(teaser, cold_stub),
         ])
 
-        district = District(
+        self.district = District(
             root_dir=self.output_dir,
             project_name=project_name,
             system_parameters=sys_params,
             coupling_graph=graph
         )
 
-        district.to_modelica()
+        self.district.to_modelica()
+
+    def test_build_teaser_single(self):
+        root_path = Path(self.district._scaffold.districts_path.files_dir).resolve()
+        assert ((root_path) / 'DistrictEnergySystem.mo').exists()
 
-        root_path = os.path.abspath(os.path.join(district._scaffold.districts_path.files_dir))
-        self.run_and_assert_in_docker(os.path.join(root_path, 'DistrictEnergySystem.mo'),
-                                      project_path=district._scaffold.project_path,
-                                      project_name=district._scaffold.project_name)
+    @pytest.mark.simulation
+    def test_simulate_teaser_single(self):
+        root_path = Path(self.district._scaffold.districts_path.files_dir).resolve()
+        self.run_and_assert_in_docker(Path(root_path) / 'DistrictEnergySystem.mo',
```

Comment:
```suggestion
        self.run_and_assert_in_docker(root_path / 'DistrictEnergySystem.mo',
```
We don't need to call Path again, but maybe it's nice to have it visible there? I'd call it personal style preference. Same comment in all the tests.